client/packages.go

```diff
@@ -172,6 +180,7 @@ func (client *Client) List(names []string, opts *ListOptions) ([]*Snap, error) {
 }
 
 // Sections returns the list of existing snap sections in the store
```

Comment:
seems many doc comments were already missing the final . in this file, something for a follow up I suppose